apps/style/curriculum/navigation.scss

```diff
@@ -0,0 +1,48 @@
+@import 'color';
+@import 'font';
+
+.nav-bar {
+  margin-right: 5px;
+  min-width: 200px;
+  font-size: 14px;
+  font-family: $gotham-extra-bold;
+
+  .category {
+    padding-left: 10px;
+    width: 200px;
+    height: 48px;
```

Comment:
I would set this to auto and the remove below `&.open` rule too. (They were functionally doing the same thing as the display none/block, but now they are redundant)